src/components/form/fields/schema-field.ts

```diff
@@ -270,6 +270,10 @@ export class SchemaField extends React.Component<FieldProps> {
     }
 
     render() {
+        if (this.props.schema.lime?.hidden) {
+            return null;
```

Comment:
> Hidden fields don't participate in CSS grid layout

Have you confirmed that that is the case?

The reason I'm asking is that I'm unsure if there will be some empty custom component rendered anyway. I see that this is a React component, so perhaps that doesn't happen? Had this been a Stencil component, returning `null` from the `render` method would still have left a `<limel-schema-field></limel-schema-field>` in the html, and that would have had to be hidden using CSS to void it taking up a space in the grid.